helixbass/tsc-rust
# Parse block comment

In this PR:
- parse block comment
- include parsing diagnostics in emitted diagnostics

To test:
If you run `cargo run tmp.tsx` against a source file containing eg:
```
/* abc
 * def */
++a
```
then you should see a diagnostic debug-logged with message `Cannot find name 'a'.`
If you run `cargo run tmp.tsx` against a source file containing eg:
```
/* a
```
then you should see a diagnostic debug-logged with message `'*/' expected.`

Based on `type-check-type-alias`
